run_tests.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+declare -a rails_versions=(5.2.5 6.0.5.1 6.1.6.1 7.0.3.1)
+
+for i in "${rails_versions[@]}"
+do
+  echo "Testing Rails Version: $i"
+
+  RAILS_VERSION=$i bundle update
+  RAILS_VERSION=$i bundle exec rspec ./spec
+done
```

Comment:
Screenshot of multi-version Rails testing:

<img width="560" alt="Screen Shot 2022-07-13 at 11 58 34 AM" src="https://user-images.githubusercontent.com/16937/178778599-0335f558-604c-48dd-9140-fe29300b5b56.png">